plugin-templates/blocks/theme-navigation/edit.tsx

```diff
@@ -0,0 +1,105 @@
+import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
+import { PanelBody, SelectControl, Spinner } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * A global type should be declared for this package. This is a temporary workaround until we
+ * decide on where global types should be declared.
```

Comment:
we might need a `types` directory in `plugin-templates` that gets copied over to the plugin 🤔

Reply:
@mogmarsh I'm unsure what the best solution might be here. As the repo stands now, we'd essentially need [one of these](https://github.com/alleyinteractive/create-wordpress-project/blob/develop/custom.d.ts) inside the plugin.

I am unsure if the intent here is to only have one package.json and one composer.json live at the root of the project or not. This might not be the case since it would tie `create-wordpress-plugin` and `create-wordpress-theme` to `create-wordpress-project`. Perhaps @kevinfodness could shed some insights here?